src/main/java/com/leafbound/services/CartServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package com.leafbound.services;
+
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import com.leafbound.models.Carts;
+import com.leafbound.repositories.CartRepository;
+
+public class CartServiceImpl implements CartService {
+	
+	
+	
+	@Autowired
+	private CartRepository crepo;
+
+	@Override
+	public Products findProdById(int id) {
```

Comment:
This method is returning `null` which is basically not implementing the method. We want to return a product.